scripts/update-image-tags.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+RED="\033[31m"
+GREEN="\033[32m"
+YELLOW="\033[33;1m"
+CYAN="\033[36;1m"
+NC="\033[0m"
+
+# A tag name must be valid ASCII and may contain lowercase and uppercase letters,
+# digits, underscores, periods and dashes.
+image_regex='quay.io/pusher/(.*)builder:([a-zA-Z0-9\._-]+)$'
+pinned_image_regex='quay.io/pusher/(.*)builder:([a-zA-Z0-9\._-]+)\s+(.+)$'
+
+TAG="${1:-}"
+
+if [[ -z $TAG ]]; then
+  echo -e "${RED}Must supply desired tag as first argument${NC}"
+  exit 1
+fi
+
+echo -e "${CYAN}Updating images to tag '$TAG'${NC}"
+
+for f in $(ls config/jobs/*/*.yaml config/README.md); do
+  echo -e "Updating $f"
+  r="s|$image_regex|quay.io/pusher/\1builder:$TAG|g"
+  # Hack to make Linux and Mac sed work in-place
```

Comment:
All of this guff requires a working local Docker in any case.
How about we avoid this arch hack by just running the script in docker?

Reply:
I don't think these scripts require a docker do they? We are just sed'ing into files in this one and making some curls in the other one? 

I do like the idea of scripts inside docker though 🤔